generators/generator-bot-core-assistant/generators/app/templates/dialogs/BotTourDialog/language-generation/en-us/BotTourDialog.en-us.lg

```diff
@@ -188,34 +203,27 @@
 }
 ```
 
+
 # BotTourDetailsCardActions()
 - ```
 {
     "type": "ActionSet",
     "actions": [
         {
             "type": "Action.Submit",
-            "title": "\"What's next on my calendar?\"",
-            "data": {
-                "intent": "SampleAction"
-            }
+            "title": "\"Suggested action A\"",
+            "data": "A"
         },
         {
             "type": "Action.Submit",
-            "title": "\"Whats the forecast tomorrow?\"",
-            "data": {
-                "intent": "SampleAction"
-            }
+            "title": "\"Suggested action B\"",
+            "data": "B"
         },
         {
             "type": "Action.Submit",
-            "title": "\"Set a timer\"",
-            "data": {
-                "intent": "SampleAction"
-            }
+            "title": "\"Suggested action C\"",
+            "data": "C"
```

Comment:
_Ideally_ I'd prefer the Action.Submit would trigger an intent that would go to another dialog which is more realistic instead of triggering a switch statement. 

I think this is okay as a first pass to go. I think not having it going through to the unknown intent is already a lot better. Thanks Ryan!

Reply:
I agree, we can revisit the strategy for Bot Tour in R14. Thanks!